test/data/css/tests.js

```diff
@@ -230,6 +230,19 @@ exports.test_data = {
             { input: '#bla, #foo{\n\tcolor:black;{{separator_input}}\tfont-size: 12px;\n}', output: '#bla,\n#foo {\n\tcolor: black;{{separator_output}}\tfont-size: 12px;\n}' }
```

Comment:
The output here has `bla,\n#foo`.  We need to preserve newlines here, too.  So need inputs such as `#bla\n,\n\n#foo` - The newline before the comma should go away, but the other two?